infra/cifuzz/actions/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Docker image to run CIFuzz in.
+FROM debian:9.5-slim
+
+RUN apt-get update && apt-get install -y git \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg2 \
```

Comment:
and this?

Reply:
Yes it is also needed to install docker.